MariaKarpava/units-tracking-app
# Daily limit view

Changes made:
- implemented the Daily Limit view to align with the Figma design specifications
- implement MVVM to establish connection of the view with the view model
- established communication with the goalsService through Notification Center
- implemented a disabled state for both the save and decrement buttons


Test Plan: 
- change Limits on DailyLimit Screen: verify that changes to the limits for units per day and per 7 days persist even after restarting the app
- after modifying the limits on the DailyLimit screen, navigate to the Home screen and confirm that its state reflects the updated number of remaining units
- check save button state when changing units: make it enabled only when there are unsaved changes. I.e. 8 -> 9 -> 8 - state should be disabled.
- check decrement button state: it should be disabled if units <= 0.
